src/par_iter/util.rs

```diff
@@ -1,25 +0,0 @@
-use std::marker::PhantomData;
-
-/// A little structure used to mark some type. This is often used when
-/// we have some phantom type we wish to remember as part of a struct.
-/// This doesn't represent "reachable data", and hence it is
-/// considered sendable regardless of `T`, unlike `PhantomData`.
-pub struct PhantomType<T> {
-    data: PhantomData<T>
-}
```

Comment:
I don't know that it was a *subversion*, but I'm glad to see it go, because it was annoying and made the code seem complex. I actually found the process of removing it quite enlightening, it suggested principles for when to choose an associated type vs a type parameter on traits that I hadn't fully understood.